site/components/ChangeLanguageButton/index.tsx

```diff
@@ -46,14 +46,15 @@ export const ChangeLanguageButton: FC = () => {
 
   return (
     <Tippy
+      className={styles.tooltip}
       content={content}
+      interactive={true}
+      onClickOutside={() => setShowMenu(false)}
       placement="bottom-end"
       visible={showMenu}
-      interactive
-      className={styles.tooltip}
     >
       <button
-        onClick={() => setShowMenu((s) => !s)}
+        onClick={() => setShowMenu(!showMenu)}
```

Comment:
in (rare) cases the closure value can be stale, hence why I always use the callback fn pattern which always has the latest state value